lib/ddtrace/contrib/sinatra/tracer.rb

```diff
@@ -41,6 +43,12 @@ def render(engine, data, *)
                   # If data is a string, it is a literal template and we don't
                   # want to record it.
                   span.set_tag(Ext::TAG_TEMPLATE_NAME, data) if data.is_a? Symbol
+
+                  # Set analytics sample rate
+                  if Contrib::Analytics.enabled?(Datadog.configuration[:sinatra][:analytics_enabled])
```

Comment:
We probably don't need this one, we likely only need the `Ext::SPAN_REQUEST` span to have the tag.

Reply:
Removed this.